### ISSUE TITLE
Incorrect tooltip position(tooltip.positioner) when tooltip.outside = true.

### ISSUE THREAD

**Issue by mateuszkornecki** [highcharts/highcharts#14002]
#### Expected behaviour
The following positioner should equally position tooltip regardless of the `tooltip.outside` property. 
```javascript
    positioner(labelWidth, labelHeight, point) {
      return {
        x: point.plotX,
        y: point.plotY,
      };
    }
```

#### Actual behaviour
When `tooltip.outside` = `true` the position is absolute to the main container instead of chart one.

#### Live demo with steps to reproduce
Compare tooltip of both charts.
https://jsfiddle.net/BlackLabel/p3jtxq1h/

#### Workaround
While working with the `tooltip.outside` enabled, include the offsets in the calculations.
```javascript
positioner(labelWidth, labelHeight, point) {
      return {
        x: point.plotX + chart.container.offsetLeft,
        y: point.plotY + chart.container.offsetTop,
      };
    }
```
**Live demo:**
https://jsfiddle.net/BlackLabel/vxmjhopr/
#### Product version
8.1.2

#### Affected browser(s)
All

**Comment by pawelfus** [comment]
That's correct @mateuszkornecki - tooltip is positioned in relation to the page, not to the container.

See our official demo how to calculate this offset: https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/tooltip/positioner-scrollable-plotarea/

**Comment by sebastianbochan** [comment]
We have a discussion with Mateusz about that. 

From one perspective its true, but from on the other hand would be great to have consistency of using formatter, when the outside param is true. Its sort of user-friendly solution.

@TorsteinHonsi what do you think?
